src/Veldrid/Vk/VulkanUtil.cs

```diff
@@ -185,6 +185,20 @@ private static bool TryLoadVulkan()
                 srcStageFlags = VkPipelineStageFlags.Transfer;
                 dstStageFlags = VkPipelineStageFlags.FragmentShader;
             }
+            else if (oldLayout == VkImageLayout.General && newLayout == VkImageLayout.TransferDstOptimal)
```

Comment:
One of these blocks isn't necessary anymore; I noticed it's been fixed in master since.

Reply:
Please lend your expertise as to what the flag values should be in the one that's still missing.  I tried to make a guess based on the rest of the cases, but I have no idea if it's correct.

Reply:
`VkImageLayout.General` should indicate that the resource was being used in a compute shader, so I'd expect srcStageFlags to be `VkPipelineStageFlags.Compute` and srcAccessmask to be something like `VkAccessFlags.ShaderWrite`

A lot of the transitions in here are... iffy. Really, this whole function should be refactored, but it's extremely tricky at this point.

Reply:
Updated, please take another look and confirm it's correct now.